tests/zopen_check_abanner

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+#
+# Basic 'zopen build' banner
+#
+# Just enables easy spotting of switch from build to test mode.
+# The name is chosen so that it shows up first in test list.
+#
+
+WORKDIR="$1"
```

Comment:
I wonder if this should be moved to https://github.com/ZOSOpenTools/metaport/blob/main/buildenv#L79?

Reply:
Got me.  there are two banners. One is the generic which shows the start.  The other has a specific name for a specific test.   They could be included or not based upon usage.  ie, you want to see when tests begin, use the base one.  Want to see a specific test?  Use the second variant with the test name in the banner text.

Reply:
That makes sense.  I’ll try to refit.  Many thanks!